src/Form/Extension/Channel/ChannelTypeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMailchimpPlugin\Form\Extension\Channel;
+
+use Setono\SyliusMailchimpPlugin\Model\ChannelInterface;
+use Sylius\Bundle\ChannelBundle\Form\Type\ChannelType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class ChannelTypeExtension extends AbstractTypeExtension
+{
+    public static function getExtendedTypes(): array
+    {
+        return [ChannelType::class];
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
```

Comment:
Not sure I understand what this actually does?

Reply:
This is to allow channel configuration in admin. 
![image](https://user-images.githubusercontent.com/9363039/87954086-2ad4db80-caac-11ea-8292-3c254ad0866e.png)